### PR TITLE
Added a link to the Non-Microsoft projects section

### PR DESCRIPTION
**Pull request by TanukiSharp**
As advised by a NuGet team member, I added a link in the `Non-Microsoft projects` section, to a minimal implementation of a NuGet server, self-hostable on any OS thanks to .NET Core.

### CODE REVIEW

File: NuGet.Docs/ndocs/Policies/Ecosystem.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ Many other individuals and companies have made significant contributions to the
 * [Sonatype Nexus](http://www.sonatype.com/nexus-repository-sonatype)
 * [SymbolSource](http://www.symbolsource.org/Public)
 * [Xamarin and MonoDevelop](https://github.com/mrward/monodevelop-nuget-addin)
+* [MinimalNugetServer](https://github.com/TanukiSharp/MinimalNugetServer)
```
Comment (karann-msft) [inline NuGet.Docs/ndocs/Policies/Ecosystem.md]: Please reorder to be in the right position according to alphabetical sorting.

**Comment by TanukiSharp** [comment]
I've fixed the issue, sorry about that.

**Comment by karann-msft** [comment]
@TanukiSharp thanks for your contribution!

**Comment by TanukiSharp** [comment]
Thanks @karann-msft for your time.

**Pull request merged**